Funcky.Test/Sequence/RepeatRangeTest.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using System.Linq;
+using FsCheck;
+using FsCheck.Xunit;
+using static Funcky.Functional;
+
+namespace Funcky.Test
+{
+    public class RepeatRangeTest
+    {
+        [Property]
+        public Property TheLengthOfTheGeneratedRepeatRangeIsCorrect(List<int> list, NonNegativeInt count)
+        {
+            var sequence = Sequence
+                .RepeatRange(list, count.Get)
+                .ToList();
+
+            return (sequence.Count == list.Count * count.Get).ToProperty();
+        }
+
+        [Property]
+        public Property AllSubsequentSubListsWithOffsetModuloCountAreIdenticalToTheGivenList(List<int> list, NonNegativeInt count)
+        {
+            var sequence = Sequence
+                .RepeatRange(list, count.Get)
+                .ToList();
+
+            return Enumerable
+                .Range(0, count.Get)
+                .Aggregate(true, (b, i)
+                    => b && sequence
+                    .Skip(i * list.Count)
+                    .Zip(list, (l, r) => l == r)
```

Comment:
I think it would improve readability if the check is done in `.All` instead of in `.Zip`

Reply:
I initially had it that way, but the Zip returns tuples.
The problem is, The Zip must create a single object (i.e tuple) and this cannot be deconstructed directly in a function signature. So we have to work with t.First and t.Second or some other alternative. Which makes it IMHO harder to read.